plugins/amdgpu/amdgpu_plugin_util.c

```diff
@@ -193,10 +193,10 @@ void print_kfd_bo_stat(int bo_cnt, struct kfd_criu_bo_bucket *bo_list)
 	for (int idx = 0; idx < bo_cnt; idx++) {
 		bo = &bo_list[idx];
 		pr_info("\n");
-		pr_info("%s(), %d. KFD BO Addr: %llx \n", __func__, idx, bo->addr);
-		pr_info("%s(), %d. KFD BO Size: %llx \n", __func__, idx, bo->size);
-		pr_info("%s(), %d. KFD BO Offset: %llx \n", __func__, idx, bo->offset);
-		pr_info("%s(), %d. KFD BO Restored Offset: %llx \n", __func__, idx, bo->restored_offset);
+		pr_info("%s(), %d. KFD BO Addr: %llx \n", __func__, idx, (unsigned long long)bo->addr);
```

Comment:
should we use PRIx64 in this case?

Reply:
`PRIx64` seems to be equivalent to `%lx` on the aarch64, ppc64 and mips64el architectures.
```
amdgpu_plugin_util.h:24:20: error: format '%lx' expects argument of type 'long unsigned int', but argument 5 has type 'long long unsigned int' [-Werror=format=]
   24 | #define LOG_PREFIX "amdgpu_plugin: "
      |                    ^~~~~~~~~~~~~~~~~
../../criu/include/log.h:47:52: note: in expansion of macro 'LOG_PREFIX'
   47 | #define pr_info(fmt, ...) print_on_level(LOG_INFO, LOG_PREFIX fmt, ##__VA_ARGS__)
      |                                                    ^~~~~~~~~~
amdgpu_plugin_util.c:198:17: note: in expansion of macro 'pr_info'
  198 |                 pr_info("%s(), %d. KFD BO Offset: %" PRIx64 "\n", __func__, idx, (unsigned long long)bo->offset);
```

Reply:
because it has to be:
pr_info("%s(), %d. KFD BO Offset: %" PRIx64 "\n", __func__, idx, bo->offset);

Reply:
```
amdgpu_plugin_util.h:24:20: error: format '%lx' expects argument of type 'long unsigned int', but argument 5 has type '__u64' {aka 'long long unsigned int'} [-Werror=format=]
   24 | #define LOG_PREFIX "amdgpu_plugin: "
      |                    ^~~~~~~~~~~~~~~~~
../../criu/include/log.h:47:52: note: in expansion of macro 'LOG_PREFIX'
   47 | #define pr_info(fmt, ...) print_on_level(LOG_INFO, LOG_PREFIX fmt, ##__VA_ARGS__)
      |                                                    ^~~~~~~~~~
amdgpu_plugin_util.c:198:17: note: in expansion of macro 'pr_info'
  198 |                 pr_info("%s(), %d. KFD BO Offset: %" PRIx64 "\n", __func__, idx, bo->offset);
      |                 ^~~~~~~
```
https://github.com/rst0git/criu/actions/runs/10019074745/job/27695225397

Reply:
https://github.com/avagin/criu/commit/cae703bae41d17e3a529e1c4d5d3d1858df60042

Reply:
@rst0git this is the trial change, I think we can make it.

Reply:
I have updated the pull request to include the changes from https://github.com/avagin/criu/commit/cae703bae41d17e3a529e1c4d5d3d1858df60042. I can confirm that the checkpoint/restore functionality works as expected with these changes on RHEL 9.4 (kernel version `5.14.0-427.28.1.el9_4.x86_64`) with the Radeon RX 7900 XTX.